plugins/agentic-docs/skills/component-docs/scripts/validate.sh

```
@@ -273,6 +273,75 @@ fi
+        clean_path=$(echo "$skip_path" | sed 's/[`*]//g' | xargs)
+        if [ -n "$clean_path" ] && [[ "$clean_path" != vendor* ]] && [[ "$clean_path" != go.* ]]; then
+            base_dir=$(echo "$clean_path" | cut -d'/' -f1)
+            if [ -d "$REPO_PATH/$base_dir" ] || [ -f "$REPO_PATH/$base_dir" ]; then
+                if [ "${VERBOSE:-false}" = "true" ]; then
+                    echo "  ✅ Skip path base exists: $base_dir"
+                fi
+            else
+                echo "  ⚠️  Skip path base not found: $base_dir (from $skip_path)"
+            fi
+        fi
+    done < <(grep -oP '`[^`]+\*\*[^`]*`' "$REPO_PATH/REVIEW.md" 2>/dev/null || true)
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)